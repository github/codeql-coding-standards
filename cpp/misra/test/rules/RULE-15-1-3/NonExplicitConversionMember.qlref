rules/RULE-15-1-3/NonExplicitConversionMember.ql