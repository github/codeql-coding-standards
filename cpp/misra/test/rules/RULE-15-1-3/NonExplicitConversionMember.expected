| test.cpp:5:3:5:12 | TestClass1 | Constructor 'TestClass1' that is callable with a single argument shall be explicit. |
| test.cpp:6:3:6:23 | operator std::int32_t | Conversion operator shall be explicit. |
| test.cpp:23:3:23:12 | TestClass4 | Constructor 'TestClass4' that is callable with a single argument shall be explicit. |
| test.cpp:25:3:25:12 | TestClass4 | Constructor 'TestClass4' that is callable with a single argument shall be explicit. |
| test.cpp:44:3:44:17 | operator double | Conversion operator shall be explicit. |
| test.cpp:45:3:45:23 | operator const char * | Conversion operator shall be explicit. |
